app-emulation/spice/patches/spice-0.15.0.patchset

```diff
@@ -0,0 +1,110 @@
+From 4844b9ad94f23a31d3bf1bb89b03ef78c480aa81 Mon Sep 17 00:00:00 2001
+From: Han Pengfei <pengphei@qq.com>
+Date: Sun, 16 Oct 2022 01:21:46 +0000
+Subject: fix meson build
+
+
+diff --git a/meson.build b/meson.build
+index fa3cfe5..09de360 100644
+--- a/meson.build
++++ b/meson.build
+@@ -104,9 +104,11 @@ foreach dep : ['libjpeg', 'zlib']
+ endforeach
+ 
+ if host_machine.system() != 'windows'
+-  foreach dep : ['rt', 'm']
+-    spice_server_deps += compiler.find_library(dep)
+-  endforeach
++	if host_machine.system() != 'haiku'
++  		foreach dep : ['rt', 'm']
++    		spice_server_deps += compiler.find_library(dep)
++  		endforeach
++  	endif
+ else
+   foreach dep : ['ws2_32', 'shlwapi']
+     spice_server_deps += compiler.find_library(dep)
+diff --git a/subprojects/spice-common/meson_options.txt b/subprojects/spice-common/meson_options.txt
+index 9f10a63..ce38053 100644
+--- a/subprojects/spice-common/meson_options.txt
++++ b/subprojects/spice-common/meson_options.txt
+@@ -28,7 +28,7 @@ option('smartcard',
+ 
+ option('python-checks',
+     type : 'boolean',
+-    value : true,
++    value : false,
```

Comment:
I think these options can be disabled in BUILD()?

Reply:
1. I have tried enabling "python-checks", but it will fail when checking **pyparsing_python3**
2. disable **python-checks** in **meson.build** instead of **meson_options.txt**